src/CMakeLists.txt

```diff
@@ -203,6 +203,25 @@ if (OpenMP_FOUND)
   target_link_libraries(dbcsr PRIVATE OpenMP::OpenMP_Fortran)
 endif ()
 
+# =================================================================================================
+# DBCSR LIBRARY's OPENCL BACKEND
+
+if (USE_OPENCL)
+  target_compile_definitions(dbcsr PRIVATE __DBCSR_ACC)
+  target_link_libraries(dbcsr PUBLIC ${OpenCL_LIBRARY})
```

Comment:
you probably want `PRIVATE`

Reply:
I just copied it from the CUDA part and thought it solved some problem for them. I will try PRIVATE.

Reply:
hmm, I remember there was a reason it was `PUBLIC` on CUDA, but not why

Reply:
I am using `PRIVATE` now; seems to be OK.